pdata/internal/common.go

```diff
@@ -262,10 +268,18 @@ func (v Value) SliceVal() Slice {
 // If the Type() is not ValueTypeBytes then returns false.
 // Calling this function on zero-initialized Value will cause a panic.
 // Modifying the returned []byte in-place is forbidden.
+// Deprecated: [0.54.0] Use BytesVal() instead.
 func (v Value) MBytesVal() []byte {
 	return v.orig.GetBytesValue()
 }
 
+// BytesVal returns the ImmutableByteSlice value associated with this Value.
+// If the Type() is not ValueTypeBytes then returns false.
```

Comment:
The comment is not accurate. This does not return bool.

Reply:
Thanks. fixed